environment-win.yml

```
@@ -24,6 +24,6 @@ dependencies:
   - pycifrw
   - pytest>=3.0
   - pytest-cov
-  - python<=3.8
+  - python>=3.6
```

I think we want 
```suggestion
  - python>3.6
```

To drop support.

---

good catch!